Mute fail (#16)
smeup/smeup-rpg: docs/visual_studio_code.md

# Configuring Visual Studio Code for running rpgle tests

1. Install the [RPG extension](https://marketplace.visualstudio.com/items?itemName=NielsLiisberg.RPG) by [Niels Liisberg](https://github.com/NielsLiisberg)

2. Install the [Code Runner extension](https://marketplace.visualstudio.com/items?itemName=formulahendry.code-runner) by [Jun Han](https://github.com/formulahendry)

> 3. [Create an executable jar](mute.md#command-line-utility) (`rpgJavaInterpreter-core-mute-all.jar`) and put it in a directory of your choice

Reviewer: ```suggestion
3. [Create an executable jar for executing MUTE tests](mute.md#command-line-utility) (`rpgJavaInterpreter-core-mute-all.jar`) and put it in a directory of your choice
```

> 4. Configure the Code Runner extension to execute this utility. An example of the portion of `settings.json` for that purpose is:

Reviewer: ```suggestion
4. Configure the Code Runner extension to execute this utility. The relevant portion of `settings.json` for that purpose is:
```

```
    "code-runner.executorMap": {
        "rpg": "cd $dir && java -DshowSourceAbsolutePath=true -jar /my/path/to/rpgJavaInterpreter-core-mute-all.jar $fileName",
    },
    "code-runner.clearPreviousOutput": true,
    "code-runner.enableAppInsights": false,
    "code-runner.saveAllFilesBeforeRun": true,
    "code-runner.showExecutionMessage": false,
    "code-runner.runInTerminal": true
```

> (Note that the `-DshowSourceAbsolutePath=true` option is for enable links to source file and line numbers.)

Reviewer: ```suggestion
_Note that the `-DshowSourceAbsolutePath=true` option is for enabling links to source file and line numbers._
```

5. Optionally configure some [snippets](https://code.visualstudio.com/docs/editor/userdefinedsnippets) to ease the editing of rpgle tests.

> For example:

Reviewer: I would add a blank line before "For example:"

```
	"MUTE equal": {
		"prefix": "mue",
		"body": [
			"MU* VAL1($1) VAL2($2) COMP(EQ)"
		],
		"description": "MUTE annotation for equality"
	}
```

> 6. Run the code with one of these methods:

Reviewer: I think that markdown requires a blank line before the bullet points

Author: Uhm, I'm not sure about this, anyway it could be more readable
